src/plugins/intel_cpu/src/emitters/x64/jit_perf_count_emitters.hpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "jit_emitter.hpp"
+#include <cpu/x64/jit_generator.hpp>
+
+// if CHRONO_CALL is defined, use std::chrono::high_resolution_clock as timer
+// otherwise uncomment below line to read tsc as cycle counters
+#define CHRONO_CALL
```

Comment:
I think we need to document somewhere these 2 available options: e.g. what limitations do we have for `tsc` approach and how differ the overheads of these two approaches. Let's discuss a suitable place for this short documentation offline
cc @IvanNovoselov

Reply:
Agree again) As a first step we need to write a brief description in the config, where the relevant field will be defined. As I already mentioned, we also need to provide an example of simple transformation that can be used by developers as a template.

Reply:
At current, description is located in config define https://github.com/openvinotoolkit/openvino/pull/19493/files#diff-0f9b324551ece4dff21169342ad00140682c372a265fa76538bdbeac5a6dc232R113-R121

Reply:
Why do we need to expose this property to plugin config? Can we keep it in LIR config instead?

Reply:
Done, thanks!